AK/Utf8View.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <AK/StringView.h>
+#include <AK/Types.h>
+
+namespace AK {
+
+class Utf8View;
+
+class Utf8CodepointIterator {
+    friend class Utf8View;
+
+public:
+    ~Utf8CodepointIterator() {}
+
+    bool operator==(const Utf8CodepointIterator&) const;
+    bool operator!=(const Utf8CodepointIterator&) const;
+    Utf8CodepointIterator& operator++();
+    u32 operator*() const;
+
+private:
+    Utf8CodepointIterator(const char*, int);
+    const unsigned char* m_ptr;
```

Comment:
I tend to feel that it's better to make a habit of consistently initializing primitive values.

Reply:
It is always initialized in the constructor; do you mean I should add `{ nullptr }` and `{ 0 }` here just to be extra sure?

Reply:
It's not the end of the world, it's just something I would consider a good habit.